test/python/result/test_mitigators.py

```diff
@@ -33,134 +35,13 @@
-                        1.000e-04,
-                        0.000e00,
-                        5.700e-03,
-                        0.000e00,
-                    ],
-                    [
-                        0.000e00,
-                        8.300e-03,
-                        3.200e-03,
-                        9.834e-01,
-                        0.000e00,
-                        0.000e00,
-                        0.000e00,
-                        5.300e-03,
-                    ],
-                    [
-                        1.170e-02,
-                        0.000e00,
-                        0.000e00,
-                        0.000e00,
-                        9.810e-01,
-                        2.500e-03,
-                        5.000e-03,
-                        0.000e00,
-                    ],
-                    [
-                        0.000e00,
-                        9.900e-03,
-                        0.000e00,
-                        0.000e00,
-                        3.900e-03,
-                        9.823e-01,
-                        0.000e00,
-                        3.500e-03,
-                    ],
-                    [
-                        0.000e00,
-                        0.000e00,
-                        1.310e-02,
-                        0.000e00,
-                        9.400e-03,
-                        1.000e-04,
-                        9.857e-01,
-                        1.200e-03,
-                    ],
-                    [
-                        0.000e00,
-                        1.000e-04,
-                        0.000e00,
-                        1.080e-02,
-                        0.000e00,
-                        9.300e-03,
-                        3.600e-03,
-                        9.899e-01,
-                    ],
-                ]
-            ),
-            "num_qubits": 3,
-            "shots": 10000,
-            "circuits": {
-                "ghz_3_qubits": {
-                    "counts_ideal": {"111": 5000, "000": 5000},
-                    "counts_noise": {
-                        "111": 4955,
-                        "000": 4886,
-                        "001": 16,
-                        "100": 46,
-                        "010": 36,
-                        "101": 23,
-                        "011": 29,
-                        "110": 9,
-                    },
-                },
-                "first_qubit_h_3_qubits": {
-                    "counts_ideal": {"000": 5000, "001": 5000},
-                    "counts_noise": {
-                        "000": 4844,
-                        "001": 4962,
-                        "100": 56,
-                        "101": 65,
-                        "011": 37,
-                        "010": 35,
-                        "110": 1,
-                    },
-                },
-            },
-        }
-    }
-
-    def compare_results(self, res1, res2):
+    rng = np.random.default_rng(42)
```

Comment:
I normally would do this as part of a `setUpClass()` just in case we ever subclass this. Using it as a class variable can lead to some weird scoping issues when subclasses come into play:

```suggestion
    @classmethod
    def setUpClass(cls):
    super().setUpClass()
    cls.rng = np.random.default_rng(42)
```

Reply:
Done, thanks.